ui/components/atoms/A010-InputWithLabel/stories.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+
+// Import index.js from the same folder
+import Input from '.'
+
+// List of stories of use cases for atom
+storiesOf('Atoms|A009 - Input', module).add('Input', () => (
```

Comment:
Please rename.. "Atoms|A009 - Input"  to "Atoms|010 - Input with label"

Reply:
I think I'd like to stick with input - its closer to the html5 nomenclature